test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
It should be an executable code block, because it’s indented and it follows a blank line. That’s the rule I came up with 😉 That’s also why list items cannot be longer than one paragraph, because there’s no way to distinguish the second paragraph of a list item from a code block.